controller/comments.go

```diff
@@ -106,19 +109,42 @@ func (c *CommentsController) Delete(ctx *app.DeleteCommentsContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
 	}
-
-	return application.Transactional(c.db, func(appl application.Application) error {
-		cm, err := appl.Comments().Load(ctx.Context, ctx.CommentID)
+	var cm *comment.Comment
+	var wi *workitem.WorkItem
+	// Following transaction verifies if a user is allowed to delete or not
+	dbErr := application.Transactional(c.db, func(appl application.Application) error {
```

Comment:
yes @hectorj2f done in https://github.com/almighty/almighty-core/pull/1377/commits/f874283fecffb00b6750783092a827e1e7363749
thanks !